pkgs/top-level/agda-packages.nix

```diff
@@ -1,17 +1,20 @@
 { pkgs, lib, callPackage, newScope, Agda }:
 
 let
+  agdaLib = import ../build-support/agda/lib.nix {
```

Comment:
If you put this into the part with all the packages then it could be inserted into everything with `callPackage`. I'm not sure which way is better

Reply:
I like your suggestion. In fact I wanted to achieve this yesterday but I did not know how.
I think I do now, after learning more about Nix. Unfortunately I had to make the set `rec`ursive. Is this a problem?
TBH Recursive sets scare me a little.